assets/aml-benchmark/components/batch_output_formatter/spec.yaml

```diff
@@ -71,7 +71,7 @@ outputs:
   unsafe_content_blocked_requests:
     type: uri_file
 code: ../src
-environment: azureml://registries/azureml/environments/model-evaluation/labels/latest
+environment: azureml://registries/azureml/environments/aoai-evaluation/labels/latest
```

Comment:
we need to change the environment name to 'evaluation'

Reply:
removed component update, renamed environment